pkg/asset/installconfig/aws/permissions.go

```diff
@@ -5,35 +5,38 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws/session"
-
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
 	ccaws "github.com/openshift/cloud-credential-operator/pkg/aws"
 	"github.com/openshift/installer/pkg/version"
 )
 
+var createNetworkingPermissions = []string{
+	// Permissions required for creating nework resources
+	"ec2:AttachInternetGateway",
+	"ec2:CreateInternetGateway",
+	"ec2:CreateNatGateway",
+	"ec2:CreateRoute",
+	"ec2:CreateRouteTable",
+	"ec2:CreateSubnet",
+	"ec2:CreateVpc",
+	"ec2:CreateVpcEndpoint",
+}
+
 var installPermissions = []string{
 	// EC2 related perms
 	"ec2:AllocateAddress",
 	"ec2:AssociateAddress",
 	"ec2:AssociateDhcpOptions",
 	"ec2:AssociateRouteTable",
-	"ec2:AttachInternetGateway",
 	"ec2:AuthorizeSecurityGroupEgress",
 	"ec2:AuthorizeSecurityGroupIngress",
 	"ec2:CopyImage",
 	"ec2:CreateDhcpOptions",
```

Comment:
this is also a candidate for createNetworkingPermissions

Reply:
also https://github.com/openshift/installer/pull/2586/files#diff-46507c958175911f28107bfe3982ea2bR68-R69